integration-tests/wallet-big-map-access-by-index.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { CONFIGS } from "./config";
+import { tokenCode, tokenInit } from "./data/tokens";
+import { MichelsonMap } from "@taquito/taquito";
+
+CONFIGS().forEach(({ lib, rpc, setup, knownBigMapContract }) => {
+  const Tezos = lib;
+
+  describe(`Test accessing big map abstraction by index using: ${rpc}`, () => {
+
+    beforeEach(async (done) => {
+      await setup()
+      done()    
+    });
+    
+    it('originates a wallet with empty bigmap and fetches the storage/bigmap', async (done) => {
```

Comment:
The test originates a contract (using the wallet API instead of the contract API). I suggest replacing the title with :
'originates a contract with empty bigmap using the wallet API and fetches the storage/bigmap'

Reply:
i made the change